extensions/wikia/ArticleVideo/bower_components/html5-skin/js/components/controlBar.js

```diff
@@ -208,6 +208,17 @@ var ControlBar = React.createClass({
     });
   },
 
+  formatSecondsWithoutLeadingZero: function (timeInSeconds) {
+    var seconds = parseInt(timeInSeconds,10) % 60;
+    var minutes = parseInt(timeInSeconds / 60, 10);
+
+    if (seconds < 10) {
+      seconds = '0' + seconds;
+    }
+
+    return minutes + ":" + seconds;
+  },
+
   populateControlBar: function() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has a complexity of 71 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)